dds_cli/__main__.py

```diff
@@ -2070,15 +2070,22 @@ def deactivate_motd(click_ctx, motd_id):
 # -- dds motd send -- #
 @motd_group_command.command(name="send")
 @click.argument("motd_id", metavar="[MOTD_ID]", nargs=1, type=int, required=True)
+@click.option(
+    "--unit-personnel-only",
+    is_flag=True,
+    required=False,
+    default=False,
+    help="Send MOTD to unit personnel only.",
+)
```

Comment:
I'd rather that this option is `--unit-only`: 
1. `--unit-personnel-only` partly sounds like it's only `Unit Personnel` who gets the email (Unit Admins should also get the email). This could cause confusion at some point. 
2. `--unit-personnel-only` is imo unnecessarily long